### DISCUSSION TITLE
Adding shadcn/ui sidebar causes react error (bug or miss-config?)

### DISCUSSION THREAD

**Discussion by bostjanpisler** [wakujs/waku#1035] [Q&A] [answered]
Added the shadcn/ui sidebar component breaks the app with the below error.
I'm using the default create waku code, other shadcn components work perfectly. 
I realize the sidebar guide is for nextjs, but curious if I can config waku or change the sidebar to make it work? Thank you

Error: 
Error: The "react" package in this environment is not configured correctly. The "react-server" condition must be enabled in any environment that runs React Server Components.

Shadcn/ui sidebar: https://ui.shadcn.com/docs/components/sidebar

**Comment by rmarscher** [comment, marked as answer]
I have the sidebar working in my waku app. Did you enable shadcn's react 19 feature that adds `"use client"` to the top of the components it installs? You should have this at the top of your sidebar.tsx file:

```
"use client";
```

**Reply by rmarscher** [reply]
This guide is useful https://ui.shadcn.com/docs/react-19

**Reply by bostjanpisler** [reply]
To fix the shadcn cli add command not adding "use client" i needed to change "rsc" to true in components.json file.
